cps/src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -226,4 +229,12 @@ public HttpResponse doGenerateSnippet(StaplerRequest req, @QueryParameter String
         }
     }
 
+    private static class StepDescriptorComparator implements Comparator<StepDescriptor>, Serializable {
```

Comment:
BTW you do not actually need to make this `Serializable` since it is only being used for rendering.